docs/reference/webhook-reference.md

```diff
@@ -0,0 +1,92 @@
+# Webhook
+
+The Webhook class enables you to view, test, and modify custom event notifications with [webhooks](webhooks.md).
+
+```python
+webhook = up42.initialize_webhook(webhook_id="d290f1ee-6c54-4b01-90e6-d701748f0851")
```

Comment:
@seedlit I updated the webhook id. Can you please approve and merge the update :)